guides/common/modules/proc_adding-a-bmc-interface.adoc

```diff
@@ -13,18 +13,28 @@ You only need the MAC address for the BMC interface if the BMC interface is mana
 ====
 
 .Procedure
-. Enable BMC on the {SmartProxy} server if it is not already enabled:
-.. Configure BMC power management on {SmartProxyServer} by running the `{foreman-installer}` script with the following options:
+. Install `ipmitool` on your {SmartProxy}:
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# {foreman-installer} --foreman-proxy-bmc=true \
---foreman-proxy-bmc-default-provider=ipmitool
+# {project-package-install} ipmitool
 ----
-.. In the {ProjectWebUI}, navigate to *Infrastructure* > *{SmartProxies}*.
-.. From the list in the *Actions* column, click *Refresh*.
-The list in the *Features* column should now include BMC.
-. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
+. Enable BMC power management on your {SmartProxy}:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# {foreman-installer} \
+--foreman-proxy-bmc-default-provider=ipmitool \
+--foreman-proxy-bmc=true
+----
+. In the {ProjectWebUI}, navigate to *Infrastructure* > *{SmartProxies}*.
+. From the list in the *Actions* column, click *Refresh*.
```

Comment:
```suggestion
. Optional: From the list in the *Actions* column, click *Refresh*.
```
This shouldn't be necessary.

Reply:
Applied, but maybe we should drop it completely. I believe Ewoud commented on another PR of mine saying that `foreman-installer` takes care of this.

Reply:
Okay, let's drop it.